runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -374,6 +375,56 @@ ir::OperandIndex BaseLoader<LoaderDomain>::loadOperand(const Tensor *tensor, ir:
   // Create TypeInfo
   ir::TypeInfo type_info(data_type, scale, zero_point);
   // Sparsity
+  loadSparsity(tensor, shape, type_info);
```

Comment:
@hseok-oh (optional) I did not make `loadSparsity` intentionally because `quantization`, which is also of `TypeInfo`'s property, did not. It would be better if `quantization` is handled in same way like:

```c++
loadQuantization();  // load scale, zero_point
loadSparsity();  // load sparsity
ir::TypeInfo type_info(data_type, scale, zero_point, sparsity); // create TypeInfo.
```

Reply:
OK. I'll check this later.